src/tutorial/runtime/index.rst

```diff
@@ -7,18 +7,19 @@ Runtime
 
 .. warning::
 
-   This section has not yet been upgraded to match Cylc 8 changes.
+   This section has been temporarily removed pending update for Cylc 8
 
 This section covers:
 
 * Associating tasks with scripts and executables.
 * Providing executables with runtime configurations.
 * Running Cylc workflows.
 
-.. toctree::
-   :name: rug-runtime-toc
-   :maxdepth: 2
+.. uncomment when upgraded:
+  .. toctree::
+     :name: rug-runtime-toc
+     :maxdepth: 2
 
-   introduction
-   runtime-configuration
-   configuration-consolidation/index
+     introduction
+     runtime-configuration
+     configuration-consolidation/index
```

Comment:
Might be easier to add warnings to these sections because removing them will break references causing build failures.

Reply:
It only generates a few warnings, and I'm not sure it's great to have a more or less unusable tutorial section in the guide ... but I'll restore with warnings for now.